redirects.ts

```diff
@@ -0,0 +1,21 @@
+import { NextApiResponse } from 'next';
+import { Store } from 'redux';
+import { GetServerSidePropsContext } from 'next-redux-wrapper';
+
+/** provids a server side redirect if you have no api key */
+export async function redirectIfNoApiKey(
```

Comment:
I added a server side redirect for this, so now if you go to /projects or /projects/id if you're not logged in it will redirect you back to the login screen.